.github/workflows/feature_7_docs.yml

```diff
@@ -1,22 +1,22 @@
-name: Develop > Tiny 6 Docs
+name: Develop > Tiny 7 Docs
 
 on:
   workflow_dispatch:
   push:
     paths:
       - '**'
     branches:
-      - 'feature/6**'
-      - 'hotfix/6**'
+      - 'feature/7**'
+      - 'hotfix/7**'
 
 jobs:
   build:
     name: Build Docs and Deploy
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14]
+        node-version: [16]
```

Comment:
is their a reason we dont use the current `stable` version of node?

Reply:
I only updated due to build issues, where the required minimum version was 16. I can update these to the current stable if its necessary.

Reply:
If the current LTS works, might as well use it. These versions bit-rot with nobody assigned to maintain them.